pwndbg/disasm/__init__.py

```diff
@@ -236,7 +234,14 @@ def near(address, instructions=1, emulate=False, show_prev_insns=True):
         # If we initialized the emulator and emulation is still enabled, we can use it
         # to figure out the next instruction.
         if emu:
-            target_candidate, size_candidate = emu.single_step()
+            # For whatever reason, the first instruction is emulated twice on
+            # unicorn-1.0.2rc1~unicorn-1.0.2rc3, but not on >= unicorn-1.0.2rc4.
+            # If the address is equal with the last one, skip it
+            last_emu_target = target_candidate
+            while last_emu_target == target_candidate:
```

Comment:
Hmm I already merged it, but just to be sure: won't this end up in an infinite loop on instruction like `jmp $` on old unicorn?

Reply:
Sorry, I didn't notice that. Fixed in #914

Reply:
Fixed in #915